app/src/validator/parser_validator.c

```diff
@@ -79,29 +80,23 @@ __Z_INLINE parser_error_t readRawVarint(parser_context_t *ctx, uint64_t *value)
     return parser_unexpected_buffer_end;
 }
 
- __Z_INLINE parser_error_t read_amino_64bits(parser_context_t *ctx, uint64_t *value) {
+__Z_INLINE parser_error_t read_amino_64bits(parser_context_t *ctx, uint64_t *value) {
     uint16_t offset = ctx->offset + ctx->lastConsumed;
 
     const uint8_t *p = ctx->buffer + offset;
     *value = 0;
 
     // Extract value
-    uint16_t shift = 0;
-    for (uint8_t i = 0; i < 8; i++, p++) {
-        const uint64_t tmp = ((*p) & 0x7Fu);
-
-        if (shift == 63 && tmp > 1) {
-            return parser_value_out_of_range;
-        }
-
-        *value += tmp << shift;
-
-        shift += 7;
+    int64_t v = 0;
```

Comment:
The code seems different here... 
from what I read, in the old code... there are only 7 bits per byte that can be used..
and there is a termination condition in the higher order bit.

In the new code, it is just expecting 8 bytes.. like a fixed64.
can we check this?